Common/font/ttffontrenderer.h

```diff
@@ -17,18 +17,23 @@
 
 #include "font/agsfontrenderer.h"
 
+#include <map>
+
+struct ALFONT_FONT;
+
 class TTFFontRenderer : public IAGSFontRenderer {
 public:
-  virtual bool LoadFromDisk(int fontNumber, int fontSize);
-  virtual void FreeMemory(int fontNumber);
-  virtual bool SupportsExtendedCharacters(int fontNumber) { return true; }
-  virtual int GetTextWidth(const char *text, int fontNumber);
-  virtual int GetTextHeight(const char *text, int fontNumber);
-  virtual void RenderText(const char *text, int fontNumber, BITMAP *destination, int x, int y, int colour) ;
-  virtual void AdjustYCoordinateForFont(int *ycoord, int fontNumber);
-  virtual void EnsureTextValidForFont(char *text, int fontNumber);
-};
+    virtual bool LoadFromDisk(int fontNumber, int fontSize) override;
```

Comment:
The project files for Windows are still restricted to Visual Studio 2008, which doesn't support the override keyword. Until the project files can be updated it may make sense to use a macro for this.